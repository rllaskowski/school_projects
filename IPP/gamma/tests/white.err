ERROR 1
ERROR 2
ERROR 3
ERROR 4
ERROR 5
ERROR 6
ERROR 7
ERROR 8
ERROR 9
ERROR 10
ERROR 14
ERROR 15
ERROR 16
ERROR 17
ERROR 24
ERROR 25
ERROR 26
ERROR 27
ERROR 34
ERROR 35
ERROR 36
ERROR 37
ERROR 39
ERROR 40
ERROR 41
ERROR 42
ERROR 52
ERROR 53
ERROR 54
ERROR 55
