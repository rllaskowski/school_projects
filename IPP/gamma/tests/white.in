 B 5 5 5 5
	B 5 5 5 5
B 5 5 5 5
B 5 5 5 5
B 5 5 5 5
 I 5 5 5 5
	I 5 5 5 5
I 5 5 5 5
I 5 5 5 5
I 5 5 5 5

B  5	5		5 	5	 

	m 1 0 0
m 1 0 1
m 1 1 0
m 1 1 1

m  1  0  0  
m	1	0	1	
m		1		1		0		
m	 	1  	1	  1

	g 1 0 0
g 1 0 1
g 1 1 0
g 1 1 1

g  1  0  0  
g	1	0	1	
g		1		1		0		
g	 	1  	1	  1

	b 1
b 1
b 1
b 1

	f 1
f 1
f 1
f 1

b  1
b	1
b	 	1

f  1
f	1
f	 	1

	q 2
q 2
q 2
q 2

q  3
q	3
q		3
q	 	3
