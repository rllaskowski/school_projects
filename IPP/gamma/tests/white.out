OK 12
1
1
1
1
0
0
0
0
4
4
4
21
21
21
1
1
1
1
